File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 2/SOAL 4/PRAK204-2410817210002-Sebastian_Abe_Santoso.py
total = []
while len(total) < 2:
    baris = input()
    total.extend(map(float, baris.split()))
r, t = total[:2]
vt = 22/7 * r * r * t
lt = 2 * 22/7 * r * t + 2 * 22/7 * r * r;
kt = 2 * 22/7 * r;
print(f"Volume = {vt:.2f}")
print(f"Luas = {lt:.2f}")
print(f"Keliling = {kt:.2f}")

File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 2/SOAL 4/PRAK204-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main(){
    float r, t;
    scanf("%f %f", &r, &t);
    float vt = 22.0/7.0 * r * r * t;
    float lt = 2.0 * 22.0/7.0 * r * t + 2 * 22.0/7.0 * r * r;
    float kt = 2.0 * 22.0/7.0 * r;
    printf("Volume = %.2f\n", vt);
    printf("Luas = %.2f\n", lt);
    printf("Keliling = %.2f\n", kt);
return 0;
}